JOBS.md

```diff
@@ -0,0 +1,55 @@
+# We're Hiring Engineers!
+
+Your primary work will be to maintain and improve Ganache, our "Ethereum Simulator". Ganache is a tool that allows developers to start a local blockchain environment to enable easy, fast, and safe Ethereum development.
+
+Ganache is packaged as a JavaScript library, a command-line application, and a Graphical User Interface. It is downloaded more than 400,000 times every month. You can check out the code here: https://github.com/trufflesuite/ganache
+
+You'll share responsibilities with your team to:
+
+ * Ship regular updates to stay up to date with the changing ecosystem
+ * Fix bugs and otherwise improve the end-user experience
+ * Implement a plug-in architecture to enable the use of user-maintained custom blockchain simulators
+ * Design and implement an interface to allow local modifications to the Ethereum Remote Procedure Call interface as well as the low-level Ethereum Virtual Machine
+ * Invent new and novel ways to improve the developer experience across our suite of tools and the rest of the Ethereum ecosystem.
+ * Implement CI/CD to fully automate our release process
+ * Write and maintain our documentation
+
+ ## About You
+
+You're comfortable with:
+ * Spoken and Written Communication
+ * Architecting solutions to open-ended problems
+ * Writing software and tests; you probably have 4+ years of professional experience
+
+Skills you'll learn, or improve upon if you're already knowledgeable:
+
+ * TypeScript and Node.js
+ * Writing high-performance JavaScript
+ * Data structures and algorithms (stacks, heaps, queues, tries, graphs, etc)
+ * Ethereum Virtual Machine
+ * Solidity
+ * Truffle
+ * Managing and interacting with an Open Source developer community
+
+## You'll have these opportunities:
+
+ * Ship software that reaches 100s of thousands of developers
+ * Connect with ConsenSys engineers from MetaMask, Infura, Pegasys, Protocol Labs, etc to coordinate and build software to further foster the Ethereum ecosystem
+ * Help build the future of economy by creating the tools used in its development
+ * Attend and speak at conferences around the world (DevCon, TruffleCon, more!)
```

Comment:
Trufflecon?

Reply:
Hopefully! Keep in mind this was all written early this year!